diagnostic_feedback/quiz.py

```diff
@@ -358,6 +357,12 @@ def save_choice(self, data, suffix=''):
         :param suffix:
         :return: response dict
         """
+        # Import is placed here to avoid model import at project startup.
+        try:
+            from submissions import api as my_api
```

Comment:
Can we call it `submission_api` or something else instead of `my_api`?

Reply:
Updated